content/en/docs/setup/platform-setup/kubesphere/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: KubeSphere Container Platform
+description: Instructions to setup a KubeSphere Container Platform for Istio.
+weight: 18
+skip_seealso: true
+keywords: [platform-setup,kubesphere,kubernetes]
+---
+
+Follow these instructions to prepare the [KubeSphere Container Platform](https://github.com/kubesphere/kubesphere) for Istio. You can download KubeSphere to easily install a Kubernetes cluster to your Linux machines.
+
+{{< tip >}}
+KubeSphere provides [All-in-One](https://kubesphere.io/docs/v2.1/en/installation/all-in-one/) and [Multi-Node](https://kubesphere.io/docs/v2.1/en/installation/multi-node/) installation. This enables quick setup and manages Kubernetes and Istio in a unified web console. This tutorial will only walk you through the All-in-One. Reference [Multi-node Installation](https://kubesphere.io/docs/v2.1/en/installation/multi-node/) for further information.
+{{< /tip >}}
+
+## Prerequisites
+
+A new Linux machine (64 bit), at least 2 CPUs and 4GB RAM, supports virtual machine or bare metal. The following operating systems are supported:
+
+- CentOS 7.4 ~ 7.7
+- Ubuntu 16.04/18.04 LTS
+- RHEL 7.4
+- Debian Stretch 9.5
+
+{{< tip >}}
+Disable the firewall or ensure your firewall meets the [port requirements](https://kubesphere.io/docs/v2.1/en/installation/port-firewall/).
+{{< /tip >}}
+
+## Provisioning Kubernetes Cluster
+
+1. Download KubeSphere to your Linux machine, it can help you to create a standard Kubernetes cluster (v1.15 by default, KubeSphere also supports v1.13 and v1.14).
+
+    {{< text bash >}}
+    $ curl -L https://kubesphere.io/download/stable/v2.1.0 > installer.tar.gz
+    $ tar -xzf installer.tar.gz
+    $ cd kubesphere-all-v2.1.0/scripts
+    {{< /text >}}
+
+1. Execute the installation script by selecting **"1) All-in-one"**:
+
+    {{< text bash >}}
+    $ ./install.sh
+    {{< /text >}}
+
+1. Installation may take 15 ~ 20 minutes. Wait until all pods are running. Access the console with the default account generated by logs.
```

Comment:
Last sentence doesn't make sense. The default account is generated by the logs? Maybe it should be something like: Access the console using the account information obtained from the installation logs.

Is the account information easy to find. I assume it's what you are showing below. Otherwise, there might need tone more info on how to obtain.

Reply:
> Last sentence doesn't make sense. The default account is generated by the logs? Maybe it should be something like: Access the console using the account information obtained from the installation logs.
> 
> Is the account information easy to find. I assume it's what you are showing below. Otherwise, there might need tone more info on how to obtain.

Yes, it's very easy to obtain from the installation logs.